perfdash/parser.go

```diff
@@ -43,6 +43,52 @@ func parseTestOutput(data []byte, buildNumber int, job string, testName string,
 			stripCount(&obj.DataItems[i])
 			result[testName].Builds[build] = append(result[testName].Builds[build], obj.DataItems[i])
 		}
+	}
+}
+
+type resourceUsagePercentiles map[string][]resourceUsages
+
+type resourceUsages struct {
+	Name   string  `json:"Name"`
+	Cpu    float64 `json:"Cpu"`
+	Memory int     `json:"Mem"`
+}
 
+type resourceUsage struct {
+	Cpu    float64
+	Memory float64
+}
+type usageAtPercentiles map[string]resourceUsage
+type podNameToUsage map[string]usageAtPercentiles
+
+func parseResourceUsageData(data []byte, buildNumber int, job string, testName string, result TestToBuildData) {
+	build := fmt.Sprintf("%d", buildNumber)
+	var obj resourceUsagePercentiles
+	if err := json.Unmarshal(data, &obj); err != nil {
+		fmt.Fprintf(os.Stderr, "error parsing JSON in build %d: %v %s\n", buildNumber, err, string(data))
+		return
+	}
+	if _, found := result[testName]; !found {
+		result[testName] = BuildData{Job: job, Version: "v1", Builds: map[string][]perftype.DataItem{}}
+	}
+	usage := make(podNameToUsage)
+	for percentile, items := range obj {
+		for _, item := range items {
+			name := RemoveDisambiguationInfixes(item.Name)
+			if _, ok := usage[name]; !ok {
+				usage[name] = make(usageAtPercentiles)
+			} // TODO(porridge): keep max usage?
+			usage[name][percentile] = resourceUsage{float64(item.Cpu), float64(item.Memory)}
```

Comment:
You keep random one, right?

Reply:
Yes, effectively the last one spit out by the test to the `.json` file.

Reply:
Max is probably more interesting as it should be more stable than a random one.

Reply:
Right, but max over which dimension? :-/

Reply:
Or should we keep a franken-record of `{max(cpu), max(memory)}` rather than `max({cpu,memory})`?

Reply:
Max(cpu), max(memory) - this one should be +- stable.

Reply:
Done, PTAL.